<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>社区医疗信息管理系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="tabPatients">
       <attribute name="title">
        <string>病人信息管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_SearchPatient">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>搜索:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchPatientEdit">
            <property name="placeholderText">
             <string>输入姓名搜索...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnAddPatient">
            <property name="text">
             <string>添加病人</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDelPatient">
            <property name="text">
             <string>删除选中</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSubmitPatient">
            <property name="text">
             <string>保存修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRevertPatient">
            <property name="text">
             <string>撤销修改</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="patientView">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDoctors">
       <attribute name="title">
        <string>医生科室管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="btnAddDoctor">
          <property name="text">
           <string>添加医生</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="doctorView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDrugs">
       <attribute name="title">
        <string>药品库存管控</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>药品库存可以直接在表格中修改</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_SearchDrug">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>搜索:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchDrugEdit">
            <property name="placeholderText">
             <string>输入药品名搜索...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="drugView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAppointments">
       <attribute name="title">
        <string>预约排班管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="btnAddAppointment">
          <property name="text">
           <string>新建预约</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="appointmentView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRecords">
       <attribute name="title">
        <string>就诊记录与处方</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPushButton" name="btnAddRecord">
          <property name="text">
           <string>新建就诊记录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="recordView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDashboard">
       <attribute name="title">
        <string>系统监控与统计</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QGroupBox" name="grpStats">
          <property name="title">
           <string>实时统计数据</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>总病人数:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="lblPatientCount">
             <property name="text">
              <string>Loading...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>在职医生:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="lblDoctorCount">
             <property name="text">
              <string>Loading...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>库存预警药品数:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="lblLowStock">
             <property name="text">
              <string>Loading...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="grpNet">
          <property name="title">
           <string>网络同步状态</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="lblSyncStatus">
             <property name="text">
              <string>等待同步...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btnExport">
            <property name="text">
             <string>导出报表 (CSV)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnBackup">
            <property name="text">
             <string>远程备份数据</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
